libs/server/Servers/GarnetServerOptions.cs

```diff
@@ -62,6 +62,11 @@ public class GarnetServerOptions : ServerOptions
         /// </summary>
         public bool EnableCluster = false;
 
+        /// <summary>
+        /// Disable epoch collision resolution algorithm.
+        /// </summary>
+        public bool DisableEpochCollision = false;
```

Comment:
this is strange to expose as option. it should be stable in main.

Reply:
I reverted this.